images/minio/config/template.apko.yaml

```diff
@@ -0,0 +1,22 @@
+contents:
+  packages:
+    - minio
+
+accounts:
+  groups:
+    - groupname: minio
+      gid: 1000
+  users:
+    - username: minio
+      uid: 1000
+  run-as: 1000
```

Comment:
did you pick this? or did it come from somewhere? not disagreeing just asking for context

Reply:
https://github.com/minio/minio/blob/master/dockerscripts/docker-entrypoint.sh
I looked at the file and could see it setup the user with the uid and gid.
I stuck with it as I searched and saw some other images do it.